src/components/GoToTopButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { memo } from "react"
+import { IoChevronUpSharp } from "react-icons/io5"
+
+import { Button } from "./Buttons"
+
+const scrollToTop = () => {
```

Comment:
@alex1092 Note that we have a `scrollIntoView.ts` inside `src/lib/utls` which we can use here instead of making a new "scrollToTop" component. Just pass `__next` as the `toId`... that will target the root Next.js div and scroll the user to the top.

ie:

```tsx
import { scrollIntoView } from "@/lib/utils/scrollIntoView"

// ...

scrollIntoView("__next")
```